nucleus/admin/monitor/src/main/java/fish/payara/admin/monitor/cli/ResetMonitoringLevels.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package fish.payara.admin.monitor.cli;
+
+import com.sun.enterprise.config.serverbeans.Config;
+import com.sun.enterprise.config.serverbeans.MonitoringService;
+import com.sun.enterprise.util.ColumnFormatter;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.inject.Inject;
+import org.glassfish.api.ActionReport;
+import org.glassfish.api.Param;
+import org.glassfish.api.admin.AdminCommand;
+import org.glassfish.api.admin.AdminCommandContext;
+import org.glassfish.api.admin.ExecuteOn;
+import org.glassfish.api.admin.RuntimeType;
+import org.glassfish.config.support.CommandTarget;
+import org.glassfish.config.support.TargetType;
+import org.glassfish.hk2.api.PerLookup;
+import org.glassfish.internal.api.Target;
+import org.jvnet.hk2.annotations.Service;
+import org.jvnet.hk2.config.ConfigSupport;
+import org.jvnet.hk2.config.TransactionFailure;
+
+/**
+ *
+ * @author Alan Roth
+ */
+@Service(name = "reset-monitoring-levels")
+@PerLookup
+@ExecuteOn({RuntimeType.DAS, RuntimeType.INSTANCE})
+@TargetType({CommandTarget.DAS, CommandTarget.STANDALONE_INSTANCE, CommandTarget.CLUSTER, CommandTarget.CONFIG})
+public class ResetMonitoringLevels implements AdminCommand {
+
+    @Param(name = "target", optional = true, defaultValue = "server-config")
+    String target;
+
+    @Param(name = "verbose", optional = true, shortName = "v")
+    private boolean verbose;
```

Comment:
So lets see if I remember... what I sure of is that using `boolean` (instead of `Boolean`) for `optional` turned out to be a bad idea. I had the same when I did my first command thinking that `false` for "not provided" would be fine. I think the issue was that the injection mechanism would act unexpected when not stating the parameter on command line.

Reply:
I'll keep that in mind in the future, should be changed now